routes/v1.js

```diff
@@ -384,7 +386,7 @@ api.declare({
 
 /** Define tasks */
 api.declare({
-  method:     'post',
+  method:     'put',
```

Comment:
... I've naughtily slipped this extra unrelated fix in ... sorry!
NB: further down in line 416 we already claim it is idempotent